libuuu/config.cpp

```diff
@@ -121,9 +123,9 @@ ConfigItem * Config::find(uint16_t vid, uint16_t pid, uint16_t ver)
 	return nullptr;
 }
 
-Config Config::find(const string &pro)
+Config Config::filter(const string &pro)
 {
-	Config items;
+	Config items{true};
```

Comment:
I'm not really satisfied with the default argument approach. An alternative would be to leave the constructor unchanged and just call a `clear` on `items` before entering into the below `for` loop to start off from scratch.